runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -626,6 +631,49 @@ private void playerClueLookup(SetMessage setMessage, String search)
 			log.warn("error looking up clues", ex);
 		}
 	}
+	
+	private void playerCombatLookup(SetMessage setMessage)
+	{
+		final HiscoreLookup lookup = getCorrectLookupFor(setMessage);
+		
+		try
+		{
+			final HiscoreResult highscores = hiscoreClient.lookup(lookup.getName(), lookup.getEndpoint());
+			
+			final String[] lvls = {"" + highscores.getAttack().getLevel(), 
+				"" + highscores.getStrength().getLevel(), 
+				"" + highscores.getDefence().getLevel(), 
+				"" + highscores.getHitpoints().getLevel(), 
+				"" + highscores.getRanged().getLevel(), 
+				"" + highscores.getPrayer().getLevel(), 
+				"" + highscores.getMagic().getLevel()};
+			final String lvlsString = String.join(" ", lvls);
+			
+			final int comblvl = client.getLocalPlayer().getCombatLevel();
```

Comment:
Shouldn't this be the combat level of the other player..?

Reply:
I thought the issue asked for a way to display your own combat info to other players

Reply:
Regardless, since writing this I've found that there's already a pull request from a while ago implementing this, so it probably makes more sense to merge that one.

Reply:
This would show everyone their own combat level each time the command is used.

Reply:
oh! okay that's pretty funny